meta-ostro/conf/layer.conf

```diff
@@ -32,14 +32,19 @@ BBFILE_PRIORITY_ostro = "6"
 # need to be increased by one each time a change is made to
 # local.conf.sample that requires manually updating a local.conf after
 # updating the meta data.
-LOCALCONF_VERSION = "3"
+#
+# A separate, Ostro OS specific variable is used to make the value
+# available to derived distros.
+OSTRO_OS_LOCALCONF_VERSION = "3"
+LOCALCONF_VERSION = "${OSTRO_OS_LOCALCONF_VERSION}"
```

Comment:
Can you elaborate more on how this would be used by a derivative distro? Depending on how much text is needed, either we can put it in here or make it a paragraph in our on-line doc.